packages/client/src/runtime/utils/SourceFileSlice.ts

```diff
@@ -19,6 +19,7 @@ export class SourceFileSlice {
   static read(filePath: string): SourceFileSlice | null {
     let content: string
     try {
+      // TODO: consider using `getSchemaPath` here?
       content = fs.readFileSync(filePath, 'utf-8')
```

Comment:
Before I start digging, do we know what `SourceFileSlice` is actually used for? Does it read schemas or what?

Reply:
It does not read schema files, it reads JS sources.
Used when we create a fancy stacktraces for errors.